### ISSUE TITLE
Different field name in `wallet.json`

### ISSUE THREAD

**Issue by hurui200320** [neow3j/neow3j#525]
When I create a wallet using neo-cli, the content is:

```
{
    // ...
    "accounts":[
        {
            // ...
            "isdefault":true,
            // ...
        }
    ],
    // ...
}
```

Where `io.neow3j.wallet.nep6.NEP6Account` defined the field name as `@JsonProperty("isDefault")`.

Which throws the exception of `com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "isdefault"`.

If I edited `wallet.json` to `isDefault`, neow3j works ok but neo-cli refuse to load.

If the typo has been made in neo-cli, and a lot of `wallet.json` has this typo, which I guess the best way to fix this is using `@JsonAlias("isdefault")`.

I'm using neo-cli v3.0.0-rc1 and neow3j 3.8.0.

**Comment by mialbu** [comment]
I'll take a look at this.

**Comment by gsmachado** [comment]
This is set to be fixed on `3.9.0` version -- which will be released pretty soon. 👍

**Comment by gsmachado** [comment]
I will close this issue since this is done (#527) and will be released soon. 👍
